.circleci/config.yml

```diff
@@ -204,6 +204,7 @@ jobs:
       - run:
           name: Setup SSH access to github
           command: |
+            mkdir -p ~/.ssh
```

Comment:
This seemed to fix the docs job in CircleCI

Reply:
Are you sure? Because I also made some changes in CircleCI configuration... might be a coincidence either way?

Reply:
Could be - all I know is the error I saw was for the directory not existing, so I created it 😉